source/renderer/app/components/widgets/forms/WalletsDropdownOption.js

```diff
@@ -4,30 +4,61 @@ import classnames from 'classnames';
 import { getColorFromRange } from '../../../utils/colors';
 import styles from './WalletsDropdownOption.scss';
 import StakePool from '../../../domains/StakePool';
+import LoadingSpinner from '../LoadingSpinner';
 
 export type WalletOption = {
   delegatedStakePool?: ?StakePool,
-  detail: string,
+  detail?: string,
   label: string,
   numberOfStakePools?: number,
   selected?: boolean,
+  syncing?: boolean,
+  syncingLabel?: string,
 };
 
 export default class WalletsDropdownOption extends Component<WalletOption> {
   renderLabelAndTicker = () => {
-    const { delegatedStakePool, label, numberOfStakePools } = this.props;
+    const {
+      delegatedStakePool,
+      label,
+      numberOfStakePools,
+      syncing,
+      syncingLabel,
+    } = this.props;
     if (!delegatedStakePool || !numberOfStakePools) {
-      return <div className={styles.label}>{label}</div>;
+      return (
+        <div className={styles.topRow}>
+          <div className={styles.topRowTicker}>
+            {syncing ? (
+              <div className={styles.label}>
+                {label}
+                <span className={styles.labelSync}> {syncingLabel}</span>
+              </div>
+            ) : (
+              <div className={styles.label}>{label}</div>
+            )}
+          </div>
+          <div className={styles.topRowSync}>
+            {syncing && (
```

Comment:
Is there a need for another `is syncing` check here?

Reply:
Yes because this check is related to loading spinner while the first check is related to label, these are 2 different elements.